Lesson2/src/main/java/pl/solvd/university/exceptions/InputException.java

```diff
@@ -0,0 +1,7 @@
+package pl.solvd.university.exceptions;
+
+public class InputException extends Exception {
```

Comment:
custom exceptions should be custom. Changing name is not smth we expect in this task.

Reply:
Done